### PR TITLE
additional tests for optional metadata fields

### PR DESCRIPTION
**Pull request by piotr-iohk**
ADP-809

 - added additional test for optional fields
 - test that no other options are supported

### CODE REVIEW

**Review by rvl — APPROVED** [review]
Nice

File: test/index.spec.js [hunk 1]
```diff
@@ -261,6 +286,10 @@ describe("cardano-metadata-submitter", () => {
 function fixture(done) {
   mkdtemp(path.join(os.tmpdir(), "cardano-metadata-submitter"))
     .then(cwd => {
+      copyTestData = function copyTestData() {
+        copySync(path.join(__dirname, "testData"), path.join(cwd, "testData"))
```
Comment (KtorZ) [inline test/index.spec.js:290]: What is `testData` ?

Comment (piotr-iohk) [reply test/index.spec.js:290]: A folder that currently has only "icon.png" file for testing `--logo` option...

Comment (KtorZ) [reply test/index.spec.js:290]: Shouldn't it be committed ?

Comment (KtorZ) [reply test/index.spec.js:290]: (ah it is at the end, I missed it!)

**Pull request merged**